src/components/Voting/VotingStatusFilter.tsx

```diff
@@ -1,26 +1,43 @@
 import { Button, Text } from '@chakra-ui/react'
 
-import { VotingProposalProps, VotingStatus } from 'models/voting'
+import {
+  VotingProposalProps,
+  VotingStatusOrOutcome,
+  VotingChoices,
+} from 'models/voting'
 
 interface VotingStatusFilterProps {
   votingProposals: VotingProposalProps[]
-  status: VotingStatus
-  setStatus: (status: VotingStatus) => void
+  status: VotingStatusOrOutcome
```

Comment:
could you maybe replace this `VotingStatusOrOutcome` with `VotingStatusChoices`? I'm seeing these warnings for that consolidated enums

![image](https://user-images.githubusercontent.com/26428605/183632116-ff3aecee-528a-4287-9418-eea069e6d461.png)